charts/timed/templates/configmap-frontend.yaml

```diff
@@ -8,3 +8,6 @@ metadata:
 data:
   TIMED_SSO_CLIENT_HOST: {{ .Values.auth.oidc.url | quote }}
   TIMED_SSO_CLIENT_ID: {{ .Values.auth.oidc.client.id | quote }}
+{{- if .Values.frontend.settings.sentry.enabled }}
+  SENTRY_DSN: {{ .Values.frontend.settings.sentry.dsn | quote }}
```

Comment:
Include other settings as well. Most importantly `tracesSampleRate`. If they are not used by the frontend yet we have to add them to the setup.

Reply:
@derrabauke done